app/views/catarse_bootstrap/projects/_dashboard_announce_expiration.html.slim

```diff
@@ -0,0 +1,20 @@
+.w-row.u-marginbottom-60
+  .w-col.w-col-1
+  .w-col.w-col-10
+    .card.card-big.card-terciary.u-radius
+      .fontsize-base= t('.explanation_01')
+      .u-text-center.fontsize-large.u-marginbottom-40.u-margintop-40
+        span.card.u-radius
+          = l(7.days.from_now.to_date)
```

Comment:
what does this hardcoded 7 means? If it should be hardcoded is better to use a constant